spec/models/pet_spec.rb

```diff
@@ -1,13 +1,14 @@
 # frozen_string_literal: true
 
 require "spec_helper"
+require "rails/generators"
 
 RSpec.describe Pet, type: :model, versioning: true do
   it "baseline test setup" do
     expect(Pet.new).to be_versioned
   end
 
-  it "can reify successfully" do
+  it "can be reified" do
```

Comment:
Oh, this is a pet peeve of mine also, the unnecessary use of the word "successful". It often shows up in flash messages, like "Changes saved successfully". Well .. yeah! 🤓